inquire/src/lib.rs

```diff
@@ -28,7 +28,7 @@
 //!   - Custom extensions for files created by [`Editor`] prompts;
 //!   - and many others!
 //!
-//! \* Date-related features are available by enabling the `date` feature.
+//! \* Date-related features are available by enabling the `date` feature and enable the `editor` feature for the text editor features
```

Comment:
Additionally, what do you think of moving this line to be placed _before_ `# Features`? It feels like the asterisks and the explanation are too far apart as well.

Reply:
I agree that is better. I will update the PR.

Reply:
I moved it per your idea. Please let me know if this works.